OTVision/dataformat.py

```diff
@@ -0,0 +1,21 @@
+METADATA: str = "metadata"
```

Comment:
This file will be extended in OP#1541 New output format. All files will be checked and strings will be replaced in OP#1541